### DISCUSSION TITLE
Редактирование уровня активности

### DISCUSSION THREAD

**Discussion by dzheykat** [LacunaHub/.github#35] [Предложения]
Было бы славно увидеть в боте возможность редактирования параметров активности участников, это сделает возможным перенос активности с других ботов или утерянных аккаунтов. Так же будет возможность вручную добавить общее значение сообщений, поскольку у меня на моем сервере на 20 тысяч сообщений больше чем зарегистрировано в Лакуне. В данный момент это можно сделать только через создание и выдачу наград, что само по себе не так удобно.

**Comment by rovius** [comment]
Учёт активности создан не для того, чтобы руками её поправлять, хотя use case с переносом данных с другого аккаунта интересный. Не думаю, что перенос с других ботов представляется возможным — их слишком много и у каждого своя модель хранения данных.
